code/game/objects/items/tools/maintenance_tools.dm

```diff
@@ -557,13 +557,17 @@
 		if(requires_superstrength_pry)
 			if(!HAS_TRAIT(user, TRAIT_SUPER_STRONG)) //basically IS_PRY_CAPABLE_CROWBAR
 				return
-		if(!attacked_door.density) //If its open
-			return
```

Comment:
This should probably just be moved after your other checks (line 571) because it now allows you to force open a powered door that is already open (doing nothing but wasting your time); whereas before with this check in place it would make nothing happen.